cli/cmd/lql.go

```diff
@@ -61,8 +61,9 @@ NOTE: This feature is not yet available!`,
 
 	// queryRunCmd represents the lql run command
 	queryRunCmd = &cobra.Command{
-		Use:   "run [query_id]",
-		Short: "run a query",
+		Aliases: []string{"execute"},
+		Use:     "run [query_id]",
```

Comment:
The standard for arguments is `<arg>` not `[arg]`:
```suggestion
		Use:     "run <query_id>",
```

Reply:
I'm pretty sure you told me that optional arguments use square brackets...